src/engine/rendergl.cpp

```diff
@@ -2337,21 +2337,29 @@ void cleanupgl()
 
 void drawskin(Texture *t, int x1, int y1, int x2, int y2, int colour, float blend, int size, const matrix4x3 *m)
 {
-    int w = max(x2-x1, 2), h = max(y2-y1, 2), tw = size ? size : t->w, th = size ? size : t->h;
-    float pw = tw*0.25f, ph = th*0.25f, qw = tw*0.5f, qh = th*0.5f, px = 0, py = 0, tx = 0, ty = 0;
-    if(w < qw)
+    int width = max(x2 - x1, 2);
```

Comment:
it'd be awesome if you could change `max` and `min` to `std::max` and `std::min` c: